.github/workflows/deploy-on-version-tag.yml

```diff
@@ -0,0 +1,57 @@
+name: Deployment
+
+on:
+  push:
+    tags:
+      - "20.*"
+      - "21.*"
+      - "22.*"
+      - "23.*"
+      - "24.*"
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: npm install
+    - run: npm test
+    - run: du -a # Log all available files # TODO: maybe remove
```

Comment:
I like `ls -ARlF` more, combined with `-I node_modules -I .git`, and any other directories that we do not want to list.

(Or `du -a --exclude='node_modules' --exclude='.git'`, whatever you prefer)

Reply:
This was only for debugging of this	action, we can drop it

Reply:
```suggestion
```